util/packager.js

```diff
@@ -1,63 +1,97 @@
-import { readdirSync, readFileSync, writeFileSync, existsSync, mkdirSync } from 'fs';
-import { sep, join } from 'path';
+import { log } from "console";
```

Comment:
It would probably make more sense for packager stuff and library updates to live as a separate PR from the Analytics updates.

Reply:
Packager and Libraries were removed yesterday. @KeeyanGhoreshi I added [CCSMCDS-364](https://jira.mitre.org/browse/CCSMCDS-364) to track work for putting that packager script and updated libraries into a new PR.